File: Programmers/codingTest_Kit/Hash/42578.cpp
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

int solution(vector<vector<string>> clothes) {
    int answer = 1;
    unordered_map<string, int> uom;
    for(auto i : clothes) uom[i[1]]++;
    for(auto i : uom) answer *= (i.second + 1);
    return answer-1;
}

File: Programmers/codingTest_Kit/Hash/42578.py
def solution(clothes):
    answer = 1
    data = dict()
    for i in clothes:
        if i[1] in data:
            data[i[1]] += 1
        else:
            data[i[1]] = 1
    for i in data.values():
        answer *= (i+1)
    return answer